reference/7.3/Microsoft.PowerShell.Utility/New-Alias.md

```diff
@@ -45,7 +45,7 @@ Get-Alias -Name "C" | Format-List *
 ```
 
 This command creates an alias named `C` to represent the `Get-ChildItem` cmdlet. It creates a
-description, quick wmi alias, for the alias and makes it read-only. The last line of the command
+description "quick gci alias", for the alias and makes it read-only. The last line of the command
```

Comment:
We should keep the leading comma for the grammar. We should wrap the value in backticks because we're talking about an exact string passed to the command.
```suggestion
description, `quick gci alias`, for the alias and makes it read-only. The last line of the command
```